### PR TITLE
Tasks version.

### PR DESCRIPTION
**Pull request by BryceEvans**

### CODE REVIEW

File: client/src/Components/Tasks/TaskListItem.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import moment from "moment";
+import { Checkbox, Label } from 'semantic-ui-react';
+import Tasks from "./Tasks";
+
+const TaskListItem = ({ task }) => {
+  return (
+    <FlexRow alignCenter justifyBetween width="full" style={{marginTop: "5px"}}>
+      <FlexColumn>
+        <Checkbox label={task.description} />
+        <p style={{ paddingTop: "10px", paddingLeft: "25px" }}>{task.assignedTo.firstName} {task.assignedTo.lastName} - {moment(task.endDate).format("MM/DD")}</p>
```
Comment (mglittleton) [inline client/src/Components/Tasks/TaskListItem.js]: This line is trying to display task.assigned.firstName and task.assigned.lastName, but "assigned" is not part of the task state.

**Review by mglittleton — COMMENTED** [review]
It looks like there's just that one error of "assigned" not being in your state for the tasks. Let me know when you're finished looking into that.

File: client/src/Components/Tasks/TaskListItem.js (hunk 1 above)
Comment (LBWright) [reply client/src/Components/Tasks/TaskListItem.js]: @mglittleton can you explain what you mean by 'task state'?

Comment (BryceEvans) [reply client/src/Components/Tasks/TaskListItem.js]: @mglittleton I just took out those fields for now. Does that still return errors for you? Once I get this merged into development I can use it for reference if I try displaying tasks another way...

Comment (mglittleton) [reply client/src/Components/Tasks/TaskListItem.js]: @LBWright , I just meant the part of the store under "tasks". It has an array of tasks with most of the info (property, start and end dates, etc), but is missing the assignedTo field for some reason.

Comment (mglittleton) [reply client/src/Components/Tasks/TaskListItem.js]: @BryceEvans it's rendering everything fine now without the names. Question though, wouldn't you only want to see the tasks that are assigned to the user? There should be no reason for that to be part of the task list itself.

**Review by mglittleton — APPROVED** [review]

**Pull request merged**